integration_tests/expr_05.py

```diff
@@ -14,7 +14,7 @@ def main0():
     assert test_multiply(a, b) == -50
     i: i64
     i = 1
-    i += 1
+    i += int(1)
```

Comment:
We'll have to figure out how to create i64 vs i32 in some systematic way, but for now this is fine.